config/locales/honors_college_thesis.en.yml

```diff
@@ -5,4 +5,4 @@ en:
     select_type:
       honors_college_thesis:
         name:               "Honors College Thesis"
-        description:        "University Honor's College Theses"
+        description:        "Theses from the University Honor's College"
```

Comment:
'Theses' should be singular here to match with the :name value. Also can you drop the apostrophe in 'Honor's College' here?